### DISCUSSION TITLE
Update via chocolatey fails due to checksum error

### DISCUSSION THREAD

**Discussion by Roemer** [iterate-ch/cyberduck#14490] [Q&A]
Hello all
I tried to update via chocolatey to `8.5.7.39535` but the update fails when checking the checksum. I then manually downloaded the file `Cyberduck-Installer-8.5.7.39535.exe` and generated the sha256 checksum with 7zip and it shows `e268855ba0a65baa294c8ffdedf1020ccd528f5a3086b9fc1445872c5eba56e8` whereas the chocolatey package expects the checksum `28d7700cf0a1bda4ecc6ab54069acb1217525398f3480972044805080559bdf7` so it is indeed different or is there some kind of problem on my side?

**Comment by dkocher** [comment]
Relates to [iterate.ch/cyberduck.io#1](https://github.com/iterate-ch/cyberduck.io/issues/1).
